pkg/network/usm/dump.go

```diff
@@ -66,6 +76,25 @@ func (e *ebpfProgram) dumpMapsHandler(manager *manager.Manager, mapName string,
 		for iter.Next(unsafe.Pointer(&key), unsafe.Pointer(&value)) {
 			output.WriteString(spew.Sdump(key, value))
 		}
+
+	case connectionStatesMap: // maps/connection_states (BPF_MAP_TYPE_HASH), key C.conn_tuple_t, value C.__u32
+		output.WriteString("Map: '" + mapName + "', key: 'C.conn_tuple_t', value: 'C.__u32'\n")
+		iter := currentMap.Iterate()
+		var key http.HttpConnTuple
+		var value uint32
+		for iter.Next(unsafe.Pointer(&key), unsafe.Pointer(&value)) {
+			output.WriteString(spew.Sdump(key, value))
+		}
+
+	case protocolDispatcherClassificationPrograms: // maps/dispatcher_classification_progs (BPF_MAP_TYPE_PROG_ARRAY), key C.__u32, value C.__u32
+		output.WriteString("Map: '" + mapName + "', key: 'C.__u32', value: 'C.__u32'\n")
+		iter := currentMap.Iterate()
+		var key uint32
+		var value uint32
+		for iter.Next(unsafe.Pointer(&key), unsafe.Pointer(&value)) {
+			output.WriteString(spew.Sdump(key, value))
+		}
```

Comment:
That's a static map, a.k.a, it won't change during our execution of the program
I don't mind having it, but not sure if it has a real impact and value

Reply:
Removed it.